.github/workflows/unit-tests.yml

```diff
@@ -21,5 +27,19 @@ jobs:
         run: |
           npm ci
 
+      - name: 'Create env file'
+        run: |
+          ENV_FILE_PATH=./apps/authentication-service/.env
+          touch ${ENV_FILE_PATH}
+          
+          # Vars
+          echo -e "${{ needs.setup-env.outputs.env_vars }}" >> ${ENV_FILE_PATH}
+          echo PORT="8089" >> ${ENV_FILE_PATH}
+          
+          # Secrets (can't load these from another job, due to GH security features)
+          echo AUTH0_CLIENT_SECRET=${{ secrets.AUTH0_CLIENT_SECRET }} >> ${ENV_FILE_PATH}
+
+          cat ${ENV_FILE_PATH}
+
       - name: Run Jest Tests
-        run: npm run test
+        run: npm run test --workspace apps/authentication-service
```

Comment:
Unless you're going to make more actions to separate the jest tests for each project this needs to run the tests for all the apps.

Reply:
Oh yeah. This was a mistake. I'll change it back.